tests/txpool/pool_test.go

```diff
@@ -167,3 +166,151 @@ func TestSimpleLocalTxLatencyBenchmark(t *testing.T) {
 	averageLatency = averageLatency / time.Duration(txToSendCount)
 	fmt.Println("Avg latency:", averageLatency)
 }
+
+// Topology of the network:
+// p2p_helper ---> node1 ---> node2 <-- RPC txns
+//
+// This test sends transaction to node2 RPC which means they are remote for node1 and local for node2
+// P2P helper is binded to node1 port, that's why we measure performance of remote txs processing
+func TestSimpleRemoteTxThroughputBenchmark(t *testing.T) {
+	t.Skip()
+
+	nonce := 0
+
+	txToSendCount := 15000
+	measureAtEvery := 1000
+
+	p2p := helper.NewP2P(fmt.Sprintf("http://%s/", rpcAddressNode1))
+
+	gotTxCh, errCh, err := p2p.Connect()
+	require.NoError(t, err)
+
+	start := time.Now()
+
+	// sender part
+	go func() {
```

Comment:
For `throughput benchmarks`: 
- Plz take a look how in `stdlib` benchmarked `mutex`: `BenchmarkMutexUncontended`, `BenchmarkMutex`, `BenchmarkMutexNoSpin`, `BenchmarkMutexSpin`. 
- they using `b.RunParallel` method - probably (up to you) better use it in this benchmark.